src/Orleans/Streams/PersistentStreams/StreamQueueBalancerType.cs

```diff
@@ -1,19 +1,22 @@
 namespace Orleans.Streams
 {
-    public enum StreamQueueBalancerType
+    /// <summary>
+    /// Built-in stream queue balancer type which is supported natively in orleans
+    /// </summary>
+    public static class BuiltInStreamQueueBalancerType
```

Comment:
but it is not a breaking change if we change this class's name back. If we want to avoid breaking change, then we need to change the class name back, which is what I try to say. 
People use this as `BalancerType = StreamQueueBalancerType.WhateverType. ` after this change, object in both sides of the "=" will become string, so it won't cause compile problems.